### ISSUE TITLE
[Feature requests] Some things that come to mind

### ISSUE THREAD

**Issue by Qx5** [athombv/com.nest#4]
Hi! Great app so far!

To be more flexible with flows here some suggestions:

add more tokens to all the cards:
<target temp> <actual temp><is away>
Is it possible to add a flow card in the And/IF column that contains no logic or is always true and you can add those tokens to this card? Then we would be able to use the actual temp in other flows.

new card suggestions:
IF:
- Schedule changed target temp (I would like to be notified when this changed. I know when I change it myself so the card "target temp changed" notifies me too often)
-When status changed to away
-Maybe even a "when movement is detected" (It has some kind of proximity sensor) 

AND:
-status is away

THEN:
-Set status to away

**Comment by RobinBol** [comment]
Hi there! Thanks for your suggestions so far. 

I do not completely understand what you are trying to say with the flow card in the "and" column that is always true? What are you trying to accomplish exactly? You can already use the (temperature) token provided by the "temperature changed" and "target temperature changed" flow cards to for example say out loud to which level the temperature changed.

For you new card suggestions:

The "when" card (when target temperature changed, but not manually) is not possible to implement due to the Nest API. We only get information when something changes, not how or who actually did it (program or person). Besides that, Nest doesn't allow developers to take advantage of the proximity sensor, this is not available through their API.

The "and" card (status is away condition), I had some time left so I created something like this. I've created two flow cards (status is away, status is home), do not use the invert option on these cards as there is also a third option: auto-away. Added these features to the develop branch, and it is currently being tested internally, will be released shortly.

The "then" card is unfortunately also not possible due to Nest's regulations, see the comment on this page: https://developer.nest.com/documentation/cloud/away-guide. It desires user input when changing the away status, which is not possible if done in a flow (at least not guaranteed).

Hope this clears things up!

**Comment by Qx5** [comment]
Thanks for your quick response! I was already afraid that the API might be a bit limited.

To clarify my unclear flow card suggestion:
I want Homey to use the temp tokens on other times then "when temp has changed". In example: When I arrive at home I want Homey to read the current and target temperature to me. I can not do that now because the temp tokens are only available on the "When temp has changed" flow cards (and the temp might have been constant on the time that I arrived home).
In other words: I want to use the temp variables in flows without using the now defined flow cards.

Hence my not so usual proposed solution of making a flow card with those tokens which is always rendered true. 

I hope it is a bit more clear now ;)

**Comment by RobinBol** [comment]
I get it now! Unfortunately I do not have a solution at the moment, but I think the feature you request might be a quite generic request, I will discuss this with the team to see if we can make something work!

Furthermore, I have added some more functionality to the app, "when" Away status changed to "x", and merged the two "and" flow cards into one! Will be released shortly.
